content/open-source/sigstore/policy-controller/policies/using-policy-controller-to-verify-signed-chainguard-images.md

```diff
@@ -1,10 +1,12 @@
 ---
```

Comment:
Just looking at the lefthand nav, would this doc make more sense in the `Working with Images` bucket?

Reply:
I moved there